package.json

```diff
@@ -31,7 +31,7 @@
     "info": "npm-scripts-info",
     "build": "trash dist && yarn build:main",
     "build:main": "tsc -p tsconfig.build.json",
-    "lint": "tslint --project tsconfig.json --config tslint.json",
+    "lint": "eslint . --ext .ts --ignore-pattern dist",
```

Comment:
I'd prefer if we had ignores in an [.eslintignore](https://eslint.org/docs/user-guide/configuring#eslintignore) rather than specify it in the script, just like we do for prettier

Reply:
If it was more than just ignoring `dist` then sure, but with it being such a simple rule I think that a file will just add unnecessary noise to the root level of the repo